src/jio-locale.ts

```diff
@@ -21,7 +21,7 @@ export const setLocaleFromUrl = async () => {
   const locale = url.searchParams.get('locale') || (new Intl.NumberFormat()).resolvedOptions().locale || sourceLocale;
   await setLocale(locale);
 };
-setLocaleFromUrl().catch(e => console.error(`Error loading locale: ${(e as Error).message}`));
+setLocaleFromUrl().catch(e => console.log(`Error loading locale: ${(e as Error).message}`));
```

Comment:
Unrelated to dark mode, but not showing this as error on each page load helps with debuging.

Reply:
Can you make this a seperate fix pr?